### PR TITLE
feat: add LSP7 tx presets

### PR DESCRIPTION
**Pull request by dzbo**

### CODE REVIEW

**Comment by frozeman** [comment]
Task linked: [CU-32ynj1z [Token Tx] Token transfer logics](https://app.clickup.com/t/32ynj1z)

File: src/components/endpoints/SendTransaction.vue [hunk 1]
```diff
@@ -30,7 +30,8 @@ const isPending = ref(false)
 
 const sampleData = computed<{ [key: string]: TransactionSelect[] }>(() => {
   const sampleEoA = '0x311611C9A46a192C14Ea993159a0498EDE5578aC'
-  const sampleUP = '0x4658F1Ac64486827f59E637bE9800Eb035b6f43C'
+  const sampleUP = '0x410f64cdc9c95b19659ccbd35cc5933c20489f6a'
```
Comment (Hugoo) [inline src/components/endpoints/SendTransaction.vue:33]: Let's make sure the address we put here always has a username and a profile picture.

Any reasons for changing the address here?

File: src/components/endpoints/SendTransaction.vue [hunk 2]
```diff
@@ -151,6 +161,96 @@ const sampleData = computed<{ [key: string]: TransactionSelect[] }>(() => {
         )}0000000000000000000000000000000000000000000000056bc75e2d63100000`,
       },
     ],
+    LSP7: [
+      {
+        label: '💰 Transfer: 1 token A (divisible) to EoA',
+        to: '0x314E7a56B08AF8E729612930dBAd70BB5A3575D9',
```
Comment (Hugoo) [inline src/components/endpoints/SendTransaction.vue:167]: why not using: `sampleEoA` address as defined above? - or just a named address?

File: src/components/endpoints/SendTransaction.vue [hunk 3]
```diff
@@ -151,6 +161,96 @@ const sampleData = computed<{ [key: string]: TransactionSelect[] }>(() => {
...
+        hasData: true,
+        data: `760d9bba000000000000000000000000${currentUP.substring(
+          2
+        )}000000000000000000000000${sampleEoA.substring(
+          2
+        )}0000000000000000000000000000000000000000000000000de0b6b3a7640000000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000000`,
+      },
+      {
+        label: '💰 Transfer: 1 token A (divisible) to EoA (without force)',
+        to: '0x314E7a56B08AF8E729612930dBAd70BB5A3575D9',
```
Comment (Hugoo) [inline src/components/endpoints/SendTransaction.vue:178]: I suggest to put this in a variable:

const tokenScAddress = xxx

Will be easier to understand what it is, and it is also used a couple of times below.

**Review by Hugoo — APPROVED** [review]
nice

File: src/components/endpoints/SendTransaction.vue (hunk 1 above)
Comment (dzbo) [reply src/components/endpoints/SendTransaction.vue:33]: Just tried different one while testing since I've "lost" access to previous one.

**Pull request merged**